docs/appendix/zowe-glossary.md

```diff
@@ -0,0 +1,300 @@
+# Glossary
+
+This glossary is part of a growing list of Zowe terminology.
+
+## Community
+
+* **Open Mainframe Project (OMP)**
+  
+  It is an organization which hosts and promotes development of open source software for the benefit of the IBM z mainframe community, including but not limited to z/OS. Zowe(.org) is one of several programs in this project. See [Open Mainframe Project website](https://www.openmainframeproject.org/) for more information.
+
+* **Zowe Conformance Program** 
+
+  The Zowe Support Provider Conformance Program gives vendors the ability to showcase their Zowe support competencies via well defined criteria. It is administered by the Linux Foundation and Open Mainframe Project.
+ 
+* **Squad** 
+  
+  A squad is a group of people contributing and participating in the Zowe project. Such a group owns one or more projects. Every squad is required to have a representative on the Technical Steering Committee (TSC), and participate in relevant working groups. See [Squads](https://github.com/Zowe/community/blob/master/Technical-Steering-Committee/squads.md) for more information.
+
+* **Zowe Component**
+ 
+  Zowe is a collection of both client and server code. You can install just some of Zowe, or all of it, depending on your needs. Zowe splits the major sections of the code into "Components" where each serve an important purpose. Server Components are packaged in a standardized way to include all services and plugins in one deliverable. Extensions to Zowe can also be delivered as third party Server Components. See: https://docs.Zowe.org/stable/extend/packaging-zos-extensions/#Zowe-component-manifest
+
+* **Plugin**
```

Comment:
Let's talk about this in TSC @anaxceron @balhar-jakub I think that sounds very silly. I have literally, throughout my 5 years on Zowe, have never seen the word "Plug-in" in the OMP Slack or talking with people on Git. The only place I've ever seen "Plugin-in" is documentation site text

Which is why I updated all occurrences of it to "Plugin"

Reply:
@DivergentEuropeans: I have no problem discussing a proposed style guide change w/ the TSC.

If the community wants to do away w/ the hyphenation, I think the Doc Squad would be happy to make that change. Keep in mind, though, that (all/most of?) our plug-ins include the hyphen in their proper names. 

For now, the style guide states that we hyphenate the word _plug-in_. Therefore, since this is our current, official stance on this item, we should all adhere to the practice of hyphenating the word. It's not fair to have any one individual change the rules for themselves, regardless of what their common sense suggests.

Please continue to hyphenate _plug-in_, and let me know when this discussion item is added to an upcoming TSC agenda so that I can participate in the discussion.

CC: @balhar-jakub @janan07

Reply:
I can bring it as a topic for this Thursday 09/08